config/api/routes/files.php

```diff
@@ -96,21 +99,21 @@
         }
     ],
     [
-        'pattern' => '(:all)/files/(:any)',
+        'pattern' => $pattern . '/files/(:any)',
         'method'  => 'DELETE',
         'action'  => function (string $path, string $filename) {
             return $this->file($path, $filename)->delete();
         }
     ],
     [
-        'pattern' => '(:all)/files/(:any)/name',
+        'pattern' => $pattern . '/files/(:any)/name',
         'method'  => 'PATCH',
         'action'  => function (string $path, string $filename) {
             return $this->file($path, $filename)->changeName($this->requestBody('name'));
         }
     ],
     [
-        'pattern' => 'files/search',
+        'pattern' => $pattern . '/search',
```

Comment:
`files/search` doesn't match with new pattern, am I wrong?

Reply:
Ok, that's a stupid mistake.